main.js

```diff
@@ -0,0 +1,31 @@
+var studyButton = document.querySelector('.study-button');
+var meditateButton = document.querySelector('.meditate-button');
+var exerciseButton = document.querySelector('.exercise-button');
+var buttonImages = document.querySelectorAll('.category-button-icons')
+
+//function that adds class with new style
+
+studyButton.addEventListener('click', function(event) {
+  hightlightButton(event)
+});
+
+meditateButton.addEventListener('click', function(event) {
+  hightlightButton(event)
+});
+
+exerciseButton.addEventListener('click', function(event) {
+  hightlightButton(event)
+});
+
+function hightlightButton(event) {
+  if (event.target.classList.contains('study-button')) {
+    event.target.classList.add('study-button-active');
+    buttonImages[0].src = 'assets/study-active.svg';
```

Comment:
button images repeats a lot, could refactor later using parameters and a for loop since were using index positions